pytorch_lightning/trainer/trainer.py

```diff
@@ -532,12 +537,11 @@ def train(self):
                     if met_min_epochs and met_min_steps:
                         self.train_loop.on_train_end()
                         return
-                    else:
-                        log.info(
-                            'Trainer was signaled to stop but required minimum epochs'
-                            f' ({self.min_epochs}) or minimum steps ({self.min_steps}) has'
-                            ' not been met. Training will continue...'
-                        )
+                    log.info(
```

Comment:
not related to this PR, but shall we rename this logging var as it may confuse with `self.log` for logging...

Reply:
Maybe, worth to think about it. 
I also found this cool library: https://loguru.readthedocs.io/en/stable/